### ISSUE TITLE
Example project isn't working

### ISSUE THREAD

**Issue by janczizikow** [Appboy/appboy-react-sdk#82]
Hello, 

I wanted to try out the example project included in this repo but I get an error from metro bundler:

![Screenshot_1587561668](https://user-images.githubusercontent.com/13835666/79987013-2f89f380-84ad-11ea-9a28-a6b13672e32b.png)

```
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `node_modules/react-native-appboy-sdk/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.
```

## System information: 
```
System:
    OS: macOS 10.15.4
    CPU: (8) x64 Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
    Memory: 334.67 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.16.2 - /usr/local/opt/node@10/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/opt/node@10/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 19, 23, 26, 27, 28, 29
      Build Tools: 23.0.1, 27.0.3, 28.0.1, 28.0.3, 29.0.2, 29.0.3
      System Images: android-19 | Google APIs Intel x86 Atom, android-26 | Google Play Intel x86 Atom, android-27 | Android TV Intel x86 Atom, android-27 | Intel x86 Atom, android-27 | Intel x86 Atom_64, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: ^0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-create-library: 3.1.2
```

I've tried the following steps, but it doesn't seem to solve the issue:

```
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
```

**Comment by Bucimis** [comment]
@janczizikow could you try again? We just released an update to the sample app that should make it compatible with react-native 0.60+

**Comment by janczizikow** [comment]
Hey @Bucimis thanks for the update, unfortunately I still get the same error :(

**Comment by Bucimis** [comment]
Could you try deleting `node_modules` (the top level one, not inside `AppboyProject`), re-running `yarn install`, ensuring `react-native-appboy-sdk` was updated to `1.20`, and seeing if it happens again? I also wonder if there's some caching issue going on

**Comment by janczizikow** [comment]
Hey @Bucimis it worked 👍 thank you! deleting  top level `node_modules` solved the issue
